### ISSUE TITLE
Add More Details in status page

### ISSUE THREAD

**Issue by ShadowGaming100** [louislam/uptime-kuma#2933]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

Updates of the Status Page
 
 # Add the following features

1. Show incident history 
- If *show empty incidents* is disabled hides all empty incidents, able to change this settings global or by status page
- Able to change how incidents are groups, by month or by day
- It shows only of 6 months if incidents are divided by month else if divided by day shows only of the current month
- refer below of that I am talking shown in the picture shown as an example 

![FFBB57BC-14E0-4419-A44C-7B5CC3DEDBDE](https://user-images.githubusercontent.com/81495140/225323470-b5c517be-d95f-4538-9fac-2f78c3d7862e.jpeg)

2. Below the monitor able to add graphs or chart
- Able to add specific number of graphs or charts
- Show the graph in day, week and month
- refer below of that I am talking shown in the picture 
![graphs](https://user-images.githubusercontent.com/81495140/225323175-dc688d61-2c0d-4207-8118-12bb02e4297a.jpeg)

3. monitor and bar chart updates
- update the bar chart, so when you click it it shows the time and if there is any incident it shows 
- able to add custom monitors description
- refer image below 
![3482DB8B-BA2C-4374-B804-8A545031C4D7](https://user-images.githubusercontent.com/81495140/225328555-4a677e64-acba-4060-8f51-b936e2913400.jpeg)

# Create New Page (maybe)

1. If click in a monitor and shows the following information 
- see uptime, ping, bar chart 
- Below shows a graph(user choice to add)
- show incidents of specific monitor refer to number one for more information about incidents 

2. When click in a incident 
- shows developer comments 
- Refer from the image below 
![6A30BAD8-0DF4-4D0F-B546-3792ABB62FC6](https://user-images.githubusercontent.com/81495140/225326000-8c9bed42-db93-4ea1-a811-c124462bc9b7.jpeg)

I will still update this 

### ✔️ Solution

in my case, we can update the current status page with the following things 
- Update bar chart to ping and if there is incident  incident  when you click on the square thing
- Able to add description or custom description on monitor
- Show bar charts below the monitors (user optional)
- show history or current incidents below the bar charts and able to customize how to show them

# And this may involve making a new page 
- when click in incidents shows the update plus developer comment and able to add, delete and edit commands, able to add tags like resolving, resolved 
- When click in a monitor shows information like ping, update, chart and incidents below

**Comment by ShadowGaming100** [comment]
And refer for this issue that can be added in incident page #2926

**Comment by ShadowGaming100** [comment]
I have more ideas but couldn’t add when I created this post

**Comment by ShadowGaming100** [comment]
My idea has been created by someone in #2943

**Comment by ShadowGaming100** [comment]
My idea has been created by someone in #2943

**Comment by tkbstudios** [comment]
@louislam you should definitely add that to uptime kuma for real.
The incidents history and current ping and all the other stuff mentioned here are amazing!
This will make everyone use uptime kuma for the same features than Atlassian Status Pages but because uptime kuma is free, even more people will talk about it!

**Comment by ShadowGaming100** [comment]
can add this even https://discordstatus.com/uptime

**Comment by ShadowGaming100** [comment]
> to uptime kuma for real.
> The incidents history and current ping and all the other stuff mentioned here are amazing!
> This will make everyone use uptime kuma for the same features than Atlassian Status Page

have more features coming soon for status and incidents

**Comment by CommanderStorm** [comment]
@Aetherinox 
Please refrain from posting `+1` / requests for updates things on issues, as this makes issue-management harder.
Issues are for discussing **what needs to be done how by whom**.
[We use 👍🏻 on issues to prioritise work](https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc), as always: [Pull Requests](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md) welcome.

https://github.com/louislam/uptime-kuma/pull/1253 is currently quite deadlocked, as it was created before multiple status pages were a thing, had changes to the database schema which were not discussed previously and is now a brutal rebase behind master.
See https://github.com/louislam/uptime-kuma/pull/1253#issuecomment-1409899850

If you/someone else wants to contribute a second version of this incident system, please feel free.
Also note that a +4k added lines is absolutely unreviewable. Please stick with smaller subfeatures first.

**Comment by mattholy** [comment]
incident tracking is very useful when you are telling users what are we  doing now.

like statuspage of Atlasian, or Cachet.

**Comment by tkbstudios** [comment]
Yeah exactly, this could be pretty useful for uptime kuma

**Comment by ShadowGaming100** [comment]
i am shocked my issue has not been still closed after a year

thanks for everyone loving the idea

**Comment by tkbstudios** [comment]
Marking the last comment as spam was unnecessary, the OP thanked everyone, that is NOT spam.

**Comment by CommanderStorm** [comment]
@tkbstudios
Moderation decisions are always a gray area. Especially, what is considered spam.
My measure of spam is "Is this productive (as in moving the issue forward), relevant to the issue and written in a productive tone?"

Spam marking was mostly because I did not want to respond to this passive-aggressive remark and because this said comment is not moving the issue forward.
Jea, might have been overly sensitve.. 🤷🏻‍♂️ 

> I am shocked my issue has not been still closed after a year

Longer answer:
If your issue contains 4 different major sub-issues it is unsurprising to me that it will take a longer time.

I am a bit tired of this demand-attitude:
Open source is delivered “as is” by volunteers.
This is not a megacorp which has full-time developers.
=> no guarantees (it might if another dev has the time for it, it might not) that an issue is implemented in any timeframe.
See https://github.com/louislam/uptime-kuma/pull/1253#issuecomment-1924191591 the relevant part of the discussion about this issue.

TL;DR: If you want it implemented, our contribution guide is here: https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md

**Comment by ShadowGaming100** [comment]
Shocked still this issue has some activity

**Comment by ShadowGaming100** [comment]
Thanks for making it clear the issue commanderstorm

**Comment by CommanderStorm** [comment]
We now have an incident history on the status page.
This resolves this issue, since the others are already tracked in a number of duplciate issues:
- https://github.com/louislam/uptime-kuma/issues/637
- https://github.com/louislam/uptime-kuma/issues/1984
- https://github.com/louislam/uptime-kuma/issues/1480
- https://github.com/louislam/uptime-kuma/issues/2282

If one sub-issue is not coverered, I would ask you to open a targeted issue just for this subissue instead.
10-in-1 issues are not something I can untangle well 😉 .
